esp-box/esp-box.c

```diff
@@ -56,6 +56,7 @@ static const char *TAG = "ESP-BOX";
 #endif
 
 static lv_disp_t *disp;
+static esp_lcd_touch_handle_t tp;   // LCD touch handle
```

Comment:
Not a blocker for this PR, just for the future: it is recommended to use the `s_` prefix for static variables at file scope.

(https://docs.espressif.com/projects/esp-idf/en/latest/esp32/contribute/style-guide.html#naming)

Reply:
I think, that we didn't use it in all BSPs.